2017/JULY/LTIME50/MAXOR.cpp

```diff
@@ -0,0 +1,51 @@
+/*
+	Made by:
+		``` Monal Shadi ``
+	Github: Monal5031
+	Codechef: monalshadi
+*/
+
+#include "bits/stdc++.h"
+
+using namespace std; 
+ 
```

Comment:
Remove Extra Spaces @Monal5031

Reply:
@Lucifer24 This space is also mentioned in linting convention too. Please check before pointing out.